tests/UrlBuildTest.php

```diff
@@ -9,15 +9,15 @@
 class UrlBuildTest extends TestCase
 {
     /** @test */
-    public function it_can_build_a_url_with_a_host()
+    public function it_can_build_a_url_with_a_host(): void
```

Comment:
Could you remove all void typehints on tests?

Reply:
Sure, done. ✅